src/ansys/sherlock/core/project.py

```diff
@@ -101,6 +102,8 @@ def import_odb_archive(
             Whether to process cutouts.
         guess_part_properties: bool
             Whether to guess part properties.
+        ims_stackup: bool, optional
+            Whether to generate an IMS stackup

```

Comment:
The new parameter should be included in the example, like the other optional parameters.

Reply:
Got it, I added `ims_stackup` to the example function call